ruby/recognize_grpc.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'options'
+require_relative 'request'
+require 'json'
+require_relative 'audio'
+
+options = Options.parse
+
+pcm, sample_rate, num_channels, total_frames = Audio.read_frames options
+
+request = RecognitionRequest.new(
+  api_key: ENV['VOICEKIT_API_KEY'],
```

Comment:
I found, that if voicekit keys not initialized, then occur next error.
```
in `urlsafe_decode64': undefined method `tr' for nil:NilClass (NoMethodError)
```
Can we implement checking on empty keys? And if it empty raise "human readable" exception?

Reply:
OK, just pushed it